src/Config/Registrar.php

```diff
@@ -14,6 +14,8 @@
 use CodeIgniter\Shield\Filters\SessionAuth;
 use CodeIgniter\Shield\Filters\TokenAuth;
 
+include_once __DIR__ . '/Constants.php';
```

Comment:
I don't know we should use the constant or not.
But even if we use it, I would like to remove the include_once line and the constant file.